integration/test/basic/setup.sh

```diff
@@ -10,3 +8,7 @@ kubectl delete configmap coredns -n kube-system
 kubectl delete clusterrolebinding system:coredns
 kubectl delete serviceaccount coredns -n kube-system
 kubectl delete clusterrole system:coredns
+
```

Comment:
I checked the latest build and looks like there is a problem with the serviceaccount and its not being deleted. Even though its in the script.

```
2020-11-30T21:38:46.239165861Z stdout F {"caller":"github.com/giantswarm/chart-operator/v2/service/controller/chart/resource/release/create.go:130","controller":"chart-operator-chart","event":"update","function":"ApplyCreateChange","level":"debug","loop":"10","message":"helm release `coredns-app` failed","object":"/apis/application.giantswarm.io/v1alpha1/namespaces/giantswarm/charts/coredns-app","resource":"release","stack":{"annotation":"rendered manifests contain a resource that already exists. Unable to continue with install: ServiceAccount \"coredns\" in namespace \"kube-system\" exists and cannot be imported into the current release: invalid ownership metadata; label validation error: missing key \"app.kubernetes.io/managed-by\": must be set to \"Helm\"; annotation validation error: missing key \"meta.helm.sh/release-name\": must be set to \"coredns-app\"; annotation validation error: missing key \"meta.helm.sh/release-namespace\": must be set to \"default\"","kind":"unknown","stack":[{"file":"/go/pkg/mod/github.com/giantswarm/helmclient/v2@v2.1.4/pkg/helmclient/install.go","line":52},{"file":"/go/pkg/mod/github.com/giantswarm/helmclient/v2@v2.1.4/pkg/helmclient/install.go","line":26}]},"time":"2020-11-30T21:38:46.238882+00:00","version":"856"}
```

https://circle-production-customer-artifacts.s3.amazonaws.com/picard/5d9efa15867c2e53b9a43822/5fc561177f489d73e966b6e1-0-build/artifacts/logs/kind-control-plane/containers/chart-operator-unique-56cd9f7866-w2l64_giantswarm_chart-operator-bc9ec1a54ca255df9898ec4f9d36b13487e09ede5f670257cae299649ecd4ccb.log?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Date=20201201T134347Z&X-Amz-SignedHeaders=host&X-Amz-Expires=60&X-Amz-Credential=AKIAJR3Q6CR467H7Z55A%2F20201201%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Signature=c929364844119951ecb2a33a54319eb6d47d84d11e828fe4ee867ebda49cd4bd

Also I never noticed till now that their S3 bucket is called picard. ❤️

Reply:
This was caused by me. I was testing having two coredns instances running along side of each other. Keeping the original coredns running that comes with the kind  and having chart-operator deploy coredns-app along side it. 

While trying to replace the kind coredns entirely I got a number of `Reason:       timeout pulling 'https://giantswarm.github.io/default-test-catalog/coredns-app-1.2.0-843b2e3d068805db2fd4c7e22632a09ae197f475.tgz'` errors from chart-operator. I'm still working on seeing if that can be resolved by overwriting the dns config to chart/app operator.

In the mean time, with huge help from @tfussell, we were able to get coredns-app deployed in the default ns while kind's coredns is still running. This is a possible solution. https://github.com/giantswarm/coredns-app/pull/52